appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -17,16 +18,18 @@ else{
 }
 
 function Rebase-Uri($originalURI){
-  $replace = $originalURI.IndexOf("file:///")
-  if($replace -ne  -1){
-      $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString("file:///".Length))
-      $replaceLength = $caseSensitiveUri.IndexOf($projectSlug)
-      if($replaceLength -ne -1){
-          $builder.Path = ($repoName, $repoCommit, $caseSensitiveUri.SubString($replaceLength+$projectSlug.Length+1) -join '/')
-          return $builder.ToString()
-      }
+  if($originalURI){
+    $replace = $originalURI.IndexOf("file:///")
+    if($replace -ne  -1){
+        $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString("file:///".Length))
```

Comment:
Remember, the arguments to PowerShell functions are not enclosed in parentheses (for a subtle reason that you can look up if you like). This should be:

```
$caseSensitiveUri = Get-CaseSensitivePath $originalURI.SubString("file:///".Length)
```